apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.BenchmarkingTool/Configuration.cs

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.ClientTesting;
+using System;
+using System.IO;
+
+namespace Google.Cloud.Storage.V1.BenchmarkingTool;
+
+/// <summary>
+/// Configuration for the benchmarking tool.
+/// </summary>
+internal sealed class Configuration
+{
+    internal int NumOfSamples { get; }
+    internal int DataSizeInBytes { get; }
+    internal string LocalFile { get; }
+    internal string OutputFolder { get; }
+    internal int UploadChunkSizeInBytes { get; }
+    internal int DownloadChunkSizeInBytes { get; }
+    internal string ObjectName => "obj_random_data";
+    internal string Bucket => _bucketName;
+
+    private readonly string _bucketName = IdGenerator.FromDateTime(prefix: "benchmark-");
```

Comment:
There's no need for this separate private field. It could just be:

```csharp
internal string Bucket { get; } = IdGenerator.FromDateTime(prefix: "benchmark-");
```

Reply:
I believe it'll then generate different bucket names every time getter of property ''Bucket' is called because IdGenerator.FromDateTime() always generate new ID based on current date time and we'll need to keep bucket name consistent .

Reply:
No, it won't. There's a big difference between these two:

```csharp
// Auto-implemented property with a backing field, but initialized at construction time
internal string Bucket { get; } = IdGenerator.FromDateTime(prefix: "benchmark-");

// Expression-bodied property with no backing field
internal string Bucket => IdGenerator.FromDateTime(prefix: "benchmark-");
```

Reply:
Okay, Got it! Thanks for explaining.